notebooks/amazon_forest_notebook.py

```diff
@@ -212,39 +210,40 @@
 from tensorflow.contrib.keras.api.keras.callbacks import ModelCheckpoint
 
 filepath="weights.best.hdf5"
-checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True, mode='max')
+checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True)
 
 # <markdowncell>
 
-# ## Create the neural network definition
+# ## Define and Train model
+
+# <markdowncell>
+
+# Here we define the model and begin training. 
+# 
+# Before starting the training process, you should first set a learning rate annealing optimization schedule by choosing a series of learning rates (learn_rates) with corresponding number of epochs for each (epochs_arr).
+# 
+# Alternatively, if you just want to run one training session at a fixed learning rate and num epochs you can just input one entry for each of these. 
 
 # <codecell>
 
 classifier = AmazonKerasClassifier()
 classifier.add_conv_layer(img_resize)
 classifier.add_flatten_layer()
 classifier.add_ann_layer(len(y_map))
-train_losses, val_losses, fbeta_score = classifier.train_model(x_train, y_train, learn_rate, epochs, batch_size, 
-                                                               validation_split_size=validation_split_size, 
-                                                               train_callbacks=[checkpoint])
 
-# <markdowncell>
-
-# ## Continue Training
+train_losses, val_losses = [], []
+epochs_arr = [20, 5, 5]
+learn_rates = [0.001, 0.0001, 0.00001]
```

Comment:
We could move these parameters (epoch and learn_rates) in the `Define hyperparameters` cell or change/remove the `Define hyperparameters` cell. It doesn't make much sense to have hyperparameters spread in the notebook when there is 1 place where they should be defined :)